addons/website/static/src/snippets/s_image_gallery/001.scss

```diff
@@ -0,0 +1,259 @@
+
+.o_gallery[data-vcss="001"] {
+    &.o_grid, &.o_masonry {
+        .img {
+            width: 100%;
+        }
+    }
+    &.o_grid {
+        &.o_spc-none div.row {
+            margin: 0;
+            > div {
+                padding: 0;
+            }
+        }
+        &.o_spc-small div.row > div {
+            padding: 0 5px;
+            margin-bottom: 10px;
+        }
+        &.o_spc-medium div.row > div {
+            padding: 0 10px;
+            margin-bottom: 20px;
+        }
+        &.o_spc-big div.row > div {
+            padding: 0 15px;
+            margin-bottom: 30px;
+        }
+        &.size-auto .row {
+            height: auto;
+        }
+        &.size-small .row {
+            height: 100px;
+        }
+        &.size-medium .row {
+            height: 250px;
+        }
+        &.size-big .row {
+            height: 400px;
+        }
+        &.size-small, &.size-medium, &.size-big {
+            img {
+                height: 100%;
+            }
+        }
+    }
+    &.o_masonry {
+        &.o_spc-none div.o_masonry_col {
+            padding: 0;
+            > img {
+                margin: 0 !important;
+            }
+        }
+        &.o_spc-small div.o_masonry_col {
+            padding: 0 5px;
+            > img {
+                margin-bottom: 10px !important;
+            }
+        }
+        &.o_spc-medium div.o_masonry_col {
+            padding: 0 10px;
+            > img {
+                margin-bottom: 20px !important;
+            }
+        }
+        &.o_spc-big div.o_masonry_col {
+            padding: 0 15px;
+            > img {
+                margin-bottom: 30px !important;
+            }
+        }
+    }
+    &.o_nomode {
+        &.o_spc-none .row div {
+            padding: 0;
+        }
+        &.o_spc-small .row div {
+            padding: 5px;
+        }
+        &.o_spc-medium .row div {
+            padding: 10px;
+        }
+        &.o_spc-big .row div {
+            padding: 15px;
+        }
+    }
+    &:not(.o_slideshow) {
+        img {
+            cursor: pointer;
+        }
+    }
+    &.o_slideshow {
+        .carousel {
+            .carousel-item.active,
+            .carousel-item-next,
+            .carousel-item-prev,
+            .carousel-control-next,
+            .carousel-control-prev {
+                padding-bottom: 64px;
+            }
+            ul.carousel-indicators li {
+                border: 1px solid #aaa;
+            }
+        }
+        ul.carousel-indicators {
+            height: auto;
+            padding: 0;
+            border-width: 0;
+            position: absolute;
+            bottom: 0;
+            width: 100%;
+            margin-left: 0;
+            left: 0%;
```

Comment:
Please respect the recommanded order for css properties https://github.com/odoo/odoo/wiki/SCSS-coding-guidelines